### ISSUE TITLE
Can we use other online stable diffusion tools online?

### ISSUE THREAD

**Issue by Yuhan-Ut** [dilevin/computer-graphics-csc317#49]
Can we use other online stable diffusion tools online?
It's almost impossible to do task 4 with such slow server
each image takes half an hour to generate, and during the generation, the model changes, that even with same prompt same seed it's another character. How can I make 6 image of same character that telling a story?

I have been trying this task since last night, and it just produced a completely different character. This makes me so disappointed.

I found this online, can I use it?
https://stablediffusionweb.com/#ai-image-generator

this is super fast comparing to the huggingface version we are using

**Comment by wenzhi-guo** [comment]
Yes, after discussing with the teaching team,  it is fine to use another online tool (but the online tool needs to use the same model versions as we have in our web app). The specific site above is OK to use for this assignment. Please indicate in your submission somewhere what specific site you used for this assignment.
